src/models/pang15/README.md

```diff
@@ -0,0 +1,60 @@
+# Pangolin (pang15)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Pangolin](./img/pang15.webp)
+
+The System76 Pangolin is a laptop with the following specifications:
+
+- CPU
+    - Supports AMD Ryzen (Hawk Point) Mobile CPUs
+        - [Ryzen 9 8945HS](https://www.amd.com/en/products/processors/laptop/ryzen/8000-series/amd-ryzen-9-8945hs.html)
+- BIOS
+    - Winbond W25R256JW flash chip
+        - WSON-8 form factor
+    - Programmed with System76 firmware (non-open)
+- EC
+    - ITE IT5571
+    - Programmed with non-open EC firmware
+- Graphics
+    - GPU: AMD Radeon 780M
+    - eDP display: 16.1" 2560×1600@120Hz LCD
+        - LCD panel: BOE NE160QDM-NY2 (or equivalent)
+            - Brightness: 350 nits (cd/m²)
+            - sRGB color coverage: 100%
+    - External video output: 
+        - 1x HDMI 2.0
+        - 2x DisplayPort 1.4 over USB-C
+- Memory
+    - Up to 96GB (2x48GB) dual-channel DDR5 @ 6400 MHz
```

Comment:
The tech specs on the website say 5600 Mhz.

Reply:
Corrected speed to 5600 in both locations.